.github/workflows/release.yml

```
@@ -49,6 +57,17 @@ jobs:
           if-no-files-found: error
 
   generate-wheels-matrix:
+    if: |
+      github.event_name == 'release' ||
+      (github.event_name == 'pull_request' && (
+        (
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'CI: build wheels'
+        ) ||
+        contains(github.event.pull_request.labels.*.name,
+                'CI: build wheels')
+      )
+      )
```

Alignment is a bit off
```suggestion
        )
      )
```